app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable that tracks all instances of `Animal`. Ensure that this behavior is intended, as it will include all instances of `Animal` and its subclasses.